common/status.c

```diff
@@ -98,13 +98,28 @@ static void status_io_full(enum log_level iodir,
 	status_send(take(towire_status_io(NULL, iodir, peer, who, p)));
 }
 
+static bool status_peer_io_filter_packettype(const u8 *p)
+{
+	int msg_type = fromwire_peektype(p);
+	switch (msg_type) {
+	case WIRE_PING:
+	case WIRE_PONG:
+		return true;
+	default:
+		break;
+	}
+	return false;
+
```

Comment:
```suggestion
	int msg_type = fromwire_peektype(p);
	switch (msg_type) {
	case WIRE_PING:
	case WIRE_PONG:
		return true;
	}
	return false;

```

Why not this?

Reply:
Jup, I was under the false impression that our code always has the default case, which doesn't seem to be true. So yes, your code looks better ;)